persistence-api/src/main/java/io/stargate/db/query/builder/QueryBuilderImpl.java

```diff
@@ -1283,12 +1283,11 @@ private boolean indexExists(Keyspace keyspace, String indexName) {
     query.append("ALTER TYPE ");
     query.append(keyspace.cqlName()).append('.').append(type.cqlName());
     assert !addColumns.isEmpty();
-    query.append(" ADD (");
+    query.append(" ADD ");
     query.append(
         addColumns.stream()
             .map(c -> c.cqlName() + " " + c.type().cqlDefinition())
             .collect(Collectors.joining(", ")));
-    query.append(")");
```

Comment:
Why removing the parentheses?

Reply:
The syntax was being flagged as incorrect:

`UPDATE TYPE k1.t1 ADD (name text, age int);`